libs/feature/editor/src/lib/components/record-form/form-field/form-field-open-data/form-field-open-data.component.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Component,
+  EventEmitter,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges,
+} from '@angular/core'
+import { FormControl } from '@angular/forms'
+import { CheckToggleComponent } from '@geonetwork-ui/ui/inputs'
+import { getGlobalConfig } from '@geonetwork-ui/util/app-config'
+import { TranslateModule } from '@ngx-translate/core'
+
+@Component({
+  selector: 'gn-ui-form-field-open-data',
+  templateUrl: './form-field-open-data.component.html',
+  styleUrls: ['./form-field-open-data.component.css'],
+  standalone: true,
+  imports: [CheckToggleComponent, TranslateModule],
```

Comment:
We try to create all new components with the OnPush changeDetectionStrategy, as a best practice, but I agree here it doesn't make much of a difference on performance.

Reply:
Oh I just didn't notice that you were using that strategy! I don't mind changing it, this was not on purpose, just default.